src/tests/Navbar.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Please add integration tests using React Testing Library as it's requested by previous code reviewer